--- memalloc/ptmalloc3/malloc-private.h	2022-03-08 16:59:07.293831383 +0100
+++ memalloc/ptmalloc3/malloc-private.h.pt	2022-03-08 16:59:02.940467051 +0100
@@ -38,7 +38,7 @@
 */
 #include <fcntl.h> /* for O_RDWR */
 #define MMAP_FLAGS           (MAP_PRIVATE)
-static int dev_zero_fd = -1; /* Cached file descriptor for /dev/zero. */
+extern int dev_zero_fd; /* Cached file descriptor for /dev/zero. */
 #define CALL_MMAP(s) ((dev_zero_fd < 0) ? \
            (dev_zero_fd = open("/dev/zero", O_RDWR), \
             mmap(0, (s), PROT_READ|PROT_WRITE, MMAP_FLAGS, dev_zero_fd, 0)) : \